lit-base/lambda_function.py

```diff
@@ -0,0 +1,24 @@
+from model_service import preprocess, inference, postprocess
```

Comment:
rename file to lambda_handler

Reply:
lambda_function.py is Lambda standard

Reply:
I don't see any documentation on setting that as the standard name. Would recommend giving it a meaningful name - `model_service_invoke.py` perhaps?

Reply:
Thanks for the suggestion! I insist `lambda_function` is more meaningful here for two reason:
1. It's the default option for lambda function in both Get Started and Template
2. It's more important for implementor to find the entry point of lambda function, which is defined in Lambda configuration not the lambda package, than to give the file a usage specific name